wire-ios/Wire-iOS/Sources/Authentication/AuthenticationInterfaceBuilder.swift

```diff
@@ -102,27 +100,12 @@ final class AuthenticationInterfaceBuilder {
                 )
             }
 
-            let environment: BackendEnvironment2 = if DeveloperFlag.multibackend.isOn {
-                defaultEnvironment
-            } else {
-                BackendEnvironment2(BackendEnvironment.shared)
-            }
-
-            let (rootView, bridge) = assembly.assemble(
+            let (rootView, bridge) = wireAuthenticationAssembly(
+                authenticationType: .new,
```

Comment:
Since now we create the assembly also for the reauth flow, I factored out the most part of it to reduce duplication.